python-threatexchange/threatexchange/content_type/pdf.py

```diff
@@ -4,21 +4,47 @@
 """
 Wrapper around the pdf content type.
 """
+
+from io import StringIO
```

Comment:
@schatten Note to self: verify this class is actually able to parse a PDF using a repl session.

Reply:
It may not - there's a unittest I broke that needs to be restored - I added a card in the project to come back around and fix it.